target/linux/lantiq/dts/DM200.dts

```diff
@@ -0,0 +1,208 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+
+    model = "DM200 - Netgear DM200";
+
+    chosen {
+        bootargs = "console=ttyLTQ0,115200";
+
+    };
+
+    aliases {
+        led-boot = &power_green;
+        led-failsafe = &power_amber;
+        led-running = &power_green;
+
+        led-internet = &dsl_green;
+        led-dsl = &dsl_amber;
+    };
+
+    memory@0 {
+        reg = <0x0 0x4000000>;
+    };
+
+    fpi@10000000 {
+        gpio: pinmux@E100B10 {
+            pinctrl-names = "default";
+            pinctrl-0 = <&state_default>;
+
+            state_default: pinmux {
+                mdio {
+                    lantiq,groups = "mdio";
+                    lantiq,function = "mdio";
+                };
+                conf_out {
+                    lantiq,pins = "io12", "io15";  // annexa, annexb
+                    lantiq,open-drain;
+                    lantiq,pull = <0>;
+                    lantiq,output = <1>;
+                };
+            };
+
+            pins_spi_default: pins_spi_default {
+                spi_in {
+                    lantiq,groups = "spi_di";
+                    lantiq,function = "spi";
+                };
+                spi_out {
+                    lantiq,groups = "spi_do", "spi_clk",
+                      "spi_cs4";
+                    lantiq,function = "spi";
+                    lantiq,output = <1>;
+                };
+            };
+        };
+
+        pcie@d900000 {
+            status = "disabled";
+        };
+    };
+
+    gphy-xrx200 {
+        compatible = "lantiq,phy-xrx200";
+        firmware = "lantiq/vr9_phy22f_a2x.bin";
+        phys = [ 00 01 ];
+    };
+
+    gpio-keys-polled {
+        compatible = "gpio-keys-polled";
+        #address-cells = <1>;
+        #size-cells = <0>;
+        poll-interval = <100>;
+        reset {
+            label = "reset";
+            gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+            linux,code = <KEY_RESTART>;
+        };
+    };
+
+    gpio-leds {
+        compatible = "gpio-leds";
+
+        power_amber: power_amber {
+            label = "dm200:amber:power";
+            gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+        };
+        power_green: power_green {
+            label = "dm200:green:power";
+            gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
+        };
+
+        lan_amber: lan_amber {
```

Comment:
please drop the alias (lan_amber: and so on) from leds which are not referenced in the aliases node.